gapic-common/test/gapic/call_options/retry_policy/settings_test.rb

```diff
@@ -74,4 +74,17 @@ def test_apply_defaults_wont_override_custom_values
     assert_equal 6, retry_policy.multiplier
     assert_equal 5, retry_policy.max_delay
   end
+
```

Comment:
What do you think about expanding this to cover all 16 error codes, with a separate test for each code?

Reply:
Sure, I can do that.

Reply:
Done. Also fixed the name of one of the test methods (I forgot methods have to be named `test_*`.)